code/modules/jobs/job_types/_job.dm

```diff
@@ -86,18 +86,34 @@
 				else if(!G.allowed_roles)
 					permitted = TRUE
 				else
+					to_chat(M, "<span class='warning'>Your current role does not permit you to spawn with [gear]!</span>")
 					permitted = FALSE
 
 				if(G.species_blacklist && (human.dna.species.id in G.species_blacklist))
+					to_chat(M, "<span class='warning'>Your current species does not permit you to spawn with [gear]!</span>")
 					permitted = FALSE
 
 				if(G.species_whitelist && !(human.dna.species.id in G.species_whitelist))
 					permitted = FALSE
+					if(G.unlocktype == GEAR_DONATOR) //Strip out donation items if the patreon has expired or they somehow have someone else's gear datum.
```

Comment:
Should this not be at the top? This looks like it only runs if the gear has a species whitelist flag.

Reply:
Patched. VSCODE is kinda weird with DM forced tabulation